packages/winhex/PKGBUILD

```diff
@@ -1,23 +1,24 @@
 # This file is part of BlackArch Linux ( http://blackarch.org ).
 # See COPYING for license details.
 
-pkgname='winhex'
+pkgname=winhex
 pkgver=19.7
 pkgrel=1
-pkgdesc='Hex Editor and Disk Editor'
+pkgdesc='Hex Editor and Disk Editor.'
 groups=('blackarch' 'blackarch-windows')
 arch=('any')
 url='https://www.x-ways.net/winhex/'
 depends=()
+source=('http://www.x-ways.net/winhex/winhex.zip')
+sha256sums=('15b864a81556267600fa952e62b74b845e319f651b9d1dc9d9b8eb721dfa348b')
+DLAGENTS=('http::/usr/bin/curl -A rofl -fLC - --retry 3 --retry-delay 3 -o %o %u')
 
 pkgver() {
-	curl 'https://www.x-ways.net/winhex/' | grep 'strong>WinHex ' | sed -r 's/.*WinHex ([0-9]+\.[0-9]+).*/\1/gi'
+  curl 'https://www.x-ways.net/winhex/' | grep 'strong>WinHex ' | sed -r 's/.*WinHex ([0-9]+\.[0-9]+).*/\1/gi'
 }
+
 package() {
-	tmpFile='/tmp/winhex.zip'
-	targetDir="${pkgdir}/usr/share/windows/winhex/"
-	mkdir -p "${targetDir}"
-	curl -L --output "${tmpFile}" "https://www.x-ways.net/winhex.zip"
-	unzip -d "${targetDir}" "${tmpFile}"
-	rm -rf "${tmpFile}"
+  rm -rf './winhex.zip'
+  mkdir -p "${pkgdir}/usr/share/windows/winhex/"
+  cp ./* "${pkgdir}/usr/share/windows/winhex/"
```

Comment:
This should be installed to `/opt` instead since it doesn't seem to have the source available. Sorry for not mentioning it in the first review.